deploy/docker/seqr/Dockerfile

```diff
@@ -1,60 +1,29 @@
-FROM python:3.7-buster
+FROM google/cloud-sdk:335.0.0
 
-LABEL maintainer="Broad TGG"
+ARG SEQR_REPO=https://github.com/populationgenomics/seqr
+ARG SEQR_GIT_SHA
+ARG SEQR_SERVICE_PORT
+ENV SEQR_SERVICE_PORT=$SEQR_SERVICE_PORT
+ENV PATH $PATH:/root/google-cloud-sdk/bin
+EXPOSE $SEQR_SERVICE_PORT
 
 # install commmon utilities
 RUN apt update && apt install -y --no-install-recommends \
-    bzip2 \
-    curl \
-    git \
     # install dependencies of the HaploPainter.pl script used to generate static pedigree images
-    perl \
-    build-essential \
-    libcairo2-dev \
-    libglib2.0-bin \
-    libglib2.0-0 \
-    libgtk2.0-dev \
-    libpango1.0-dev \
-    # install database clients for debugging (https://www.postgresql.org/download/linux/debian/)
-    postgresql-11 \
-    postgresql-client-11 \
-    libpq-dev \
-    redis-tools \
-    # install dev dependencies for react, javascript development. These are not needed at runtime.
-    nodejs && \
+    libgtk2-perl libdbi-perl libtk-perl libsort-naturally-perl && \
     apt clean && rm -rf /var/lib/apt/lists/* && \
-    curl -L0 https://raw.github.com/miyagawa/cpanminus/master/cpanm -o /usr/bin/cpanm && \
-    chmod +x /usr/bin/cpanm && \
-    cpanm --notest \
-    Cairo \
-    DBI \
-    Gtk2 \
-    Tk \
-    Sort::Naturally && \
-    # install gcloud tools
-    curl https://sdk.cloud.google.com | bash
-
-ENV PATH $PATH:/root/google-cloud-sdk/bin
-
-ARG SEQR_REPO=https://github.com/populationgenomics/seqr
-ARG SEQR_GIT_SHA
-
-# update seqr repo
-RUN git clone -q $SEQR_REPO seqr && \
+    # update seqr repo
+    git clone -q $SEQR_REPO seqr && \
```

Comment:
It probably doesn't matter, but this checks out the whole seqr repo with all it's history. We could consider [this process](https://www.techiedelight.com/clone-git-repository-with-specific-revision/):

```bash
# create and initialize an empty repository
git init
 
# add a remote named origin for the repository at <repository>
git remote add origin <repository>
 
# fetch a commit using its hash
git fetch origin <commit-hash>
 
# reset repository to that commit
git reset --hard FETCH_HEAD
```

Reply:
Yes, I'm not super happy with this either...

IMHO we shouldn't clone the git repository at all here, given that we're building this Dockerfile from a freshly cloned repository ;).

It might make sense to simply `COPY` the necessary files here instead, with an appropriate Docker context. Maybe something for a follow-up PR?